### PR TITLE
[EAP7-1795] Update wildfly-proposal doc

### PR DESCRIPTION
**Pull request by fl4via**
Jira: https://issues.redhat.com/browse/EAP7-1795

### CODE REVIEW

File: backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc
Comment (fl4via) [inline backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc]: @bstansberry  Is it correct to move the design notes from here to a Google document? I followed the RFE process document that mentions there is an optional design document deliverable., since there is no Design Notes in the template used by the RFE process document.

**Comment by fl4via** [comment]
@bstansberry  @ropalka please review

File: backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc
Comment (ropalka) [reply backward-compatibility/EAP7-1795_JavaEE_And_JakartaEE_Compatibility.adoc]: I'll wait for @bstansberry response to this question before approving and merging @fl4via

**Review by ropalka — APPROVED** [review]

**Pull request merged**
